igvc_navigation/src/field_d_planner/FieldDPlanner.cpp

```diff
@@ -136,10 +135,17 @@ void FieldDPlanner::publish_expanded_set(pcl::PointCloud<pcl::PointXYZRGB>& expa
       p.g = 125;
       p.b = static_cast<uint8_t>((std::get<0>(e.second) / max_g) * 255.0f);
     }
-    expanded_cloud.points.push_back(p);
+    expanded_cloud->points.push_back(p);
   }
 
-  expanded_pub_.publish(expanded_cloud);
+  // pass cloud through a VoxelGrid to reduce size
+  pcl::VoxelGrid<pcl::PointXYZRGB> sor;
```

Comment:
I'm not sure about whether we need to reduce the size of this. If you look at the rosbag for one particular bag file, `expanded` only has like 571 messages, while all 6 variants of the map pointclouds have like 2000 messages.

Reply:
I agree. I implemented this mostly for visualization tuning but it doesn't add much. I'm going to revert to 5228adb, remove some unnecessary asserts, then merge